lms/templates/content_item_selection/new_content_item_selection.html.jinja2

```diff
@@ -15,6 +15,7 @@
         <input id="url" type="text" name="document_url" onchange="contentItemSelection.resetError(this)"/>
         <span class="error"></span>
       </div>
+      {% if request.feature("new_oauth") %}<button onclick="window.open('/api/canvas/authorize?authorization=' + JSON.parse(document.querySelector('.js-config').textContent).authorization_param, 'popup', 'height=475,width=775')">Select PDF from Canvas</button>{% endif %}
```

Comment:
Strictly speaking the `authorization` param should be encoded using `encodeURIComponent` or some other method of encoding the params. I realize this works fine though because we know that the generated string only contains alphanumeric characters and a space.

Reply:
Heh. The edits to this file will be undone, as soon as we get the new UI integrated. This was just a super hacky way of shoving a button in here for testing. It should be using `request.route_url(...)` to get the `/api/canvas/authorize` URL too, instead of hardcoding that. (Or else the server needs to add that URL to the JS config object.)